packages/shared/src/components/post/PostActions.tsx

```diff
@@ -56,6 +58,21 @@ export function PostActions({
       : undefined,
   });
 
+  const { toggleBookmark } = useBookmarkPost({
```

Comment:
We just need to make double sure wherever this previous `onBookmark` came from didn't have specific tracking we need to integrate here.

Reply:
Went over the analytics that was sent before, and I can't find any difference from then to now.